packages/components/src/internal/components/domainproperties/samples/SampleTypeDesigner.tsx

```diff
@@ -55,63 +58,86 @@ const SAMPLE_SET_NAME_EXPRESSION_TOPIC = 'sampleIDs#patterns';
 const SAMPLE_SET_NAME_EXPRESSION_PLACEHOLDER = 'Enter a naming pattern (e.g., S-${now:date}-${dailySampleCount})';
 const SAMPLE_SET_HELP_TOPIC = 'createSampleType';
 
+const ALIQUOT_OPTIONS_HELP = (
+    <div>
+        <p>
+            <b>Editable for samples only:</b> Field is editable for samples but not for aliquots. An aliquot will
+            inherit the field value from its parent sample.
+        </p>
+        <p>
+            <b>Editable for aliquots only:</b> Field is viewable and editable for aliquots but not for samples.
+        </p>
+        <p>
+            <b>Separately editable for samples and aliquots:</b> Field is editable for samples and aliquots
+            independently.
+        </p>
+        <br />
+        <p>
+            Learn more about <HelpLink topic={SAMPLE_ALIQUOT_TOPIC}>Sample Aliquots</HelpLink>.
+        </p>
+    </div>
+);
+
 interface Props {
+    aliquotNamePatternProps?: AliquotNamePatternProps;
     api?: ComponentsAPIWrapper;
-    onChange?: (model: SampleTypeModel) => void;
-    onCancel: () => void;
-    onComplete: (response: DomainDesign) => void;
+    appPropertiesOnly?: boolean;
     beforeFinish?: (model: SampleTypeModel) => void;
-    initModel: DomainDetails;
+    // DomainDesigner props
```

Comment:
Previously, these props were organized into groups, but the linter has undone that, so the comments should probably go away (or else we revert the linting).